traffic_monitor/todata/todata.go

```diff
@@ -342,6 +346,48 @@ func getServerTypes(mc tc.TrafficMonitorConfigMap) (map[tc.CacheName]tc.CacheTyp
 	return serverTypes, nil
 }
 
+// getServerPartners gets the cache parners that have the same VIP
+func getServerParners(mc tc.TrafficMonitorConfigMap) map[tc.CacheName]map[tc.CacheName]bool {
+	serverPartners := map[tc.CacheName]map[tc.CacheName]bool{}
```

Comment:
Since `ServerPartners` was renamed to `SameIpServers` in some places in 2e553a404f, this should stick to the same naming scheme.

Reply:
cleaned it up